### ISSUE TITLE
Can't set application whitelist if naxsi firewall is enabled

### ISSUE THREAD

**Issue by jannispl** [deis/deis#4727]
v1.11.2
```
$ deis config:set DEIS_WHITELIST="10.0.0.0/8:internal1,172.16.0.0/12:internal2,192.168.0.0/16:internal3,100.64.0.0/10:internal4"
Creating config... Error: 
400 Bad Request
<html>
<head><title>400 Bad Request</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<hr><center>nginx/1.9.4</center>
</body>
</html>
```

```
deis-1# journalctl -fu deis-router@3
Nov 06 14:07:23 deis-1.appspark.prod sh[1335]: [INFO] - 2015/11/06 14:07:23 [error] 991#0: *5284 NAXSI_FMT: ip=46.xx.xx.xx&server=deis.example.org&uri=/v1/apps/appspark-mail/config/&learning=0&vers=0.53-1&total_processed=3&total_blocked=1&block=1&cscore0=$SQL&score0=12&zone0=BODY&id0=1015&var_name0=deis_whitelist, client: 46.252.136.90, server: ~^deis\.(?<domain>.+)$, request: "POST /v1/apps/appspark-mail/config/ HTTP/1.1", host: "deis.example.org"
Nov 06 14:07:23 deis-1.appspark.prod sh[1335]: [INFO] - [06/Nov/2015:14:07:23 +0000] - 46.252.136.90 - - - 400 - "POST /v1/apps/appspark-mail/config/ HTTP/1.1" - 323 - "-" - "Deis Client v1.11.2" - "~^deis\x5C.(?<domain>.+)$" - - - deis.example.org - - - 0.000
```

**Comment by bacongobbler** [comment]
@jannispl can you please confirm if this has been fixed in v1.12.3? We've shifted around a lot of the nginx templating code in there so hopefully application whitelists and the naxsi firewall should be able to co-exist.

**Comment by bacongobbler** [comment]
closing due to inactivity. please re-open if necessary. thanks!
